src/commands.rs

```diff
@@ -2082,7 +2079,10 @@ fn log_template(settings: &UserSettings) -> String {
 fn cmd_log(ui: &mut Ui, command: &CommandHelper, args: &LogArgs) -> Result<(), CommandError> {
     let workspace_command = command.workspace_helper(ui)?;
 
-    let revset_expression = revset::parse(&args.revisions)?;
+    let revset_expression = (match &args.revisions {
+        Some(revs) => revset::parse(revs),
+        None => revset::parse(&ui.settings().default_revset()),
```

Comment:
nit: The `revset::parse()` call could be moved out, and then the inner expression could use `.unwrap_or_else()`. I'm fine with keeping it the way it is if you prefer.

Reply:
Looks good. I think you can even replace the `unwrap_or_else(|| &default_revset)` by `unwrap_or(&default_revset)` now that there are no function calls in there.

Reply:
Done.